src/ImagineFormat.jl

```diff
@@ -224,7 +224,11 @@ const field_key_dict = Dict{AbstractString,Function}(
     "hend"                         => x->parse(Int,x),
     "vstart"                       => x->parse(Int,x),
     "vend"                         => x->parse(Int,x),
-    "angle from horizontal (deg)"  => float64_or_empty)
+    "angle from horizontal (deg)"  => float64_or_empty,
+    "x translation in pixels"      => x->parse(Int,x) != 0,
+    "y translation in pixels"      => x->parse(Int,x) != 0,
+    "rotation angle in degree"     => x->parse(Int,x) != 0)
```

Comment:
I think this should be parsed as a Float64.  @kdw503 can you confirm this?

Reply:
Yes. 'rotation angle in degree' saved as float.

Reply:
Okay, 

I changed it and committed again. I will merge the change sometime soon.